com.one_ware.OneWare.yml

```diff
@@ -0,0 +1,53 @@
+app-id: com.one_ware.OneWare
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet8
+build-options:
+  prepend-path: "/usr/lib/sdk/dotnet8/bin"
+  append-ld-library-path: "/usr/lib/sdk/dotnet8/lib"
+  env:
+    PKG_CONFIG_PATH: "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet8/lib/pkgconfig"
+  arch:
+    arm:
+      env:
+        RUNTIME: "linux-arm"
+    aarch64:
+      env:
+        RUNTIME: "linux-arm64"
+    x86_64:
+      env:
+        RUNTIME: "linux-x64"
+
+command: oneware
+separate-locales: false
+tags:
+  - proprietary
+finish-args:
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --filesystem=home
+modules:
+  - name: oneware
+    buildsystem: simple
+    build-commands:
+      - dotnet publish --source nuget-sources -c Release -r $RUNTIME -p:PublishReadyToRun=false -o ./studio ./source/studio/OneWare.Studio.Desktop/OneWare.Studio.Desktop.csproj
+      - install -D -m755 oneware ${FLATPAK_DEST}/bin/oneware
+      - install -D -m644 studio/${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -D -m644 studio/Assets/${FLATPAK_ID}.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      - install -D -m644 studio/${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - cp -r studio ${FLATPAK_DEST}/bin/studio
+    sources:
+      - type: git
+        url: https://github.com/one-ware/OneWare.git
+        disable-shallow-clone: true
+        branch: main
```

Comment:
This needs a tag or at least a commit https://docs.flatpak.org/en/latest/module-sources.html#git-sources